src/kOS.Safe/Encapsulation/Structure.cs

```diff
@@ -255,7 +255,7 @@ public virtual StringValue GetKOSInheritance()
 
         public override string ToString()
         {
-            return "Structure ";
+            return KOSNomenclature.GetKOSName(GetType()) + "()"; // print as the KOSNomenclature string name
```

Comment:
How about when we don't implement a class-specific ToString we'll just put a colon and empty quotes there with no further data?  ``GetKOSName(GetType()) + ": \"\""`` ?